qa/sc_evm_multiple_cert_submitters.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+import logging
+import os
+import time
+
+from eth_utils import remove_0x_prefix
+
+from SidechainTestFramework.account.ac_chain_setup import AccountChainSetup
+from SidechainTestFramework.account.httpCalls.wallet.balance import http_wallet_balance
+from SidechainTestFramework.account.utils import (convertZenToZennies, convertZenniesToWei)
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, MCConnectionInfo, SCCreationInfo, \
+    SC_CREATION_VERSION_2, SCNetworkConfiguration, KEY_ROTATION_CIRCUIT
+from SidechainTestFramework.sc_forging_util import check_mcreference_presence, check_mcreferencedata_presence
+from SidechainTestFramework.scutil import (
+    generate_next_block, generate_next_blocks, bootstrap_sidechain_nodes, AccountModel, connect_sc_nodes, assert_true
+)
+from test_framework.util import (
+    assert_equal, websocket_port_by_mc_node_index,
+)
+
+"""
+Check multiple certificate submitters processing for non-ceasing sidechain:
+    - Check that "faster" node which submits the certificate first, does not cause "slower" node to output error when trying to 
+    submit its certificate for the same epoch.
+
+Configuration:
+    Start 1 MC node and 2 SC nodes.
+    SC Node 1 is the only forger.
+    SC node 1 has 3 schnorr private keys [0, 1, 2] for cert submission. Submitter and signer are ENABLED.
+    SC node 2 has 3 other schnorr private keys [3, 4, 5] for cert submission. Submitter is ENABLED and signer is ENABLED.
+    Sidechain is non-ceasing sidechain.
+
+Test:
+    - creates new forward transfer to sidechain
+    - verifies that the receiving account balance is changed
+    - generate MC and SC blocks to reach the end of the Withdrawal epoch 0
+    - generate one more MC and SC block accordingly and await for certificate submission to MC node mempool
+    - check epoch 0 certificate with not backward transfers in the MC mempool
+    - check that only one SC node forged certificate successfuly, and the other one contains 'submission not needed message'
+"""
+
+
+class SCEvmBackwardTransfer(AccountChainSetup):
```

Comment:
Please rename the class in **SCEvmMultipleCertSubmittersTest**

Reply:
Done